http/1.7/modules/ROOT/pages/http-authentication.adoc

```diff
@@ -699,6 +700,229 @@ In the *Configuration XML* editor, the `tls:context`, `tls:trust-store` and `tls
+This example uses dynamic configuration represented by the use of an expression in the `resourceOwnerId` parameter. See xref:mule-sdk::static-dynamic-configs.adoc[Dynamic Configurations].
+After `</http:listener-config>` and before `<flow name="authCodeFlow">`, add the following statement to enable the OAuth authorization code:
+
+[source,xml,linenums]
+----
+<oauth:token-manager-config name="tokenManagerConfig"/>
+
+<http:request-config name="requestConfig">
+  <http:request-connection protocol="HTTPS" host="${host}" port="${port}" >
+    <http:authentication>
+      <oauth:authorization-code-grant-type
+        localCallbackConfig="listenerConfig"  
+        localCallbackConfigPath="${local.callback.path}" 
+        localAuthorizationUrl="${local.authorization.url}"         
+        authorizationUrl="${authorization.url}"
+        clientId="${connected.app.client.id}"
+        clientSecret="${connected.app.client.secret}"
+        scopes="full"
+        tokenUrl="${token.url}"
+        tokenManager="tokenManagerConfig" 
+        externalCallbackUrl="${external.callback.url}" 
+        refreshTokenWhen="#[attributes.statusCode == 401]" 
+        localAuthorizationUrlResourceOwnerId="#['yourAnypointEmail']" 
+        resourceOwnerId="#['yourAnypointEmail']" />
+    </http:authentication>
+  </http:request-connection>
+</http:request-config>
+----
+
+Add the same redirection to your YAML file:
+
+[source,YAML,linenums]
+----
+host: "anypoint.mulesoft.com"
+port: "443"
+resource: "/apiplatform/repository/v2/organizations/<your-org-id>/environments"
+local.authorization.url: "http://localhost:8070/authorize"
+authorization.url: “https://anypoint.mulesoft.com/accounts/api/v2/oauth2/authorize”
+connected.app.client.id: “???”
+connected.app.client.secret: “???”
+local.callback.path: “???”
+external.callback.url: “???”
+token.url: “???”
+----
+
+=== Add the Connected App
+
+The Connected Apps feature enables you to delegate access to applications that use APIs to interact with Anypoint Platform. This feature supports OAuth 2.0.  See xref:access-management::connected-apps-developers.adoc[Connected Apps]. To create a Connected App in the Anypoint Platform:
+
+. In Anypoint Platform > *Access Management* > select *Connected Apps*.
+. Click *Create app*.
+. Set a name for your app.
+. Set *Type* to *App acts on behalf of a user*.
+. Set *Grant Type* to *Authorization Code*.
+. Set *Website URL* to a website you choose.
+. Set *Redirect URIs* to your Mule app. You are configuring the `external.callback.url`.
+. In *Scopes*, set *General* to *Full Access*.
+. Click *Save*.
++
+A new Connected App appears. Anypoint Platform assigns a client ID and client secret.
```

Comment:
```suggestion
A new connected app appears. Anypoint Platform assigns a client ID and client secret.
```

Reply:
My rule of thumb is to use "Connected App" to refer to the feature, but "connected app" to refer to an app that you create with the Connected App feature.